packages/amplify-category-function/package.json

```diff
@@ -18,7 +18,9 @@
     "amplify-category-storage": "2.2.1",
     "fs-extra": "^8.1.0",
     "graphql-transformer-core": "6.14.1",
+    "cron-builder": "^0.3.0",
     "inquirer": "^7.0.3",
+    "inquirer-datepicker-prompt": "^0.4.2",
```

Comment:
This is an unmaintained package (2 years old), please take a look at: `inquirer-datepicker` instead.

Reply:
fixed